### PR TITLE
Add missing steps

### PR DESCRIPTION
**Pull request by Tharsanan1**

### CODE REVIEW

File: en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md [hunk 1]
```diff
@@ -60,9 +60,18 @@ The topics below explain the configurations.
...
 
-     You are navigated to the detailed configuration page. 
+     You are navigated to the detailed configuration page.
 
-5. Click and expand the **Inbound Authentication Configuration** section, click and expand **SAML2 Web SSO Configuration**, and click **Configure**.
+5. Click and expand the **Claim Configuration** section.
+
+    1. Select `Use Local Claim Dialect` checkbox.
+    2. Click on `Add Claim URI` button next to `Requested Claims:` lebel and 
+        * For `Local Claim` select `http://wso2.org/claims/groups` from the dropdown.
```
Comment (Mariangela) [inline en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md:69]:
```suggestion
        * For **Local Claim** select `http://wso2.org/claims/groups` from the dropdown.
```

File: en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md [hunk 2]
```diff
@@ -60,9 +60,18 @@ The topics below explain the configurations.
...
+        * Tick `Mandatory Claim` checkbox.
```
Comment (Mariangela) [inline en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md:70]:
```suggestion
        * Tick **Mandatory Claim** checkbox.
```

File: en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md [hunk 3]
```diff
@@ -110,7 +122,21 @@ Similarly, add the Identity Server as an identity provider configurations in `ht
     
 4. Upload the public certificate of Identity Server under **Identity Provider Public Certificate**.
 
-5. Configure **Federated authenticators** > **SAML2 Web SSO Configurations**
+5.  Click and expand the **Claim Configuration** section, and click and expand **Basic Claim Configuration**.
+    1. Tick `Define Custom Claim Dialect` radio box.
+    2. Click `Add Claim Mapping` and add the following data
```
Comment (Mariangela) [inline en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md:127]:
```suggestion
    2. Click `Add Claim Mapping` and add the following data.
```

File: en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md [hunk 4]
```diff
@@ -110,7 +122,21 @@ Similarly, add the Identity Server as an identity provider configurations in `ht
...
+        * Identity Provider Claim URI: http://wso2.org/claims/groups
```
Comment (Mariangela) [inline en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md:128]:
```suggestion
        * Identity Provider Claim URI: `http://wso2.org/claims/groups`
```

File: en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md [hunk 5]
```diff
@@ -110,7 +122,21 @@ Similarly, add the Identity Server as an identity provider configurations in `ht
...
+        * Local Claim URI: http://wso2.org/claims/roles
```
Comment (Mariangela) [inline en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md:129]:
```suggestion
        * Local Claim URI: `http://wso2.org/claims/roles`
```

File: en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md [hunk 6]
```diff
@@ -110,7 +122,21 @@ Similarly, add the Identity Server as an identity provider configurations in `ht
...
+    3. In the dropdown next to `Role Claim URI:` label select `http://wso2.org/claims/groups`
```
Comment (Mariangela) [inline en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md:130]:
```suggestion
    3. In the dropdown next to the **Role Claim URI** label, select `http://wso2.org/claims/groups`
```

File: en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md [hunk 7]
```diff
@@ -110,7 +122,21 @@ Similarly, add the Identity Server as an identity provider configurations in `ht
...
+
+6.  Click and expand the **Role Configuration** section.
+    1. Click `Add role Mapping` button.
```
Comment (Mariangela) [inline en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md:133]:
```suggestion
    1. Click **Add role Mapping**.
```

File: en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md [hunk 8]
```diff
@@ -164,7 +190,7 @@ Similarly, add the Identity Server as an identity provider configurations in `ht
     !!! note
         Note that the Publisher, Developer Portal, and Admin Portal service providers will be listed under **Service Providers** after you have logged in to each of the portals at least once.
          
-3. Click and expand **Local and Outbound Authentication Configuration**, and select Identity Provider as the **Federated Authentication**.
+3. Click and expand **Local and Outbound Authentication Configuration**, select Identity Provider as the **Federated Authentication** and tick `Assert identity using mapped local subject identifier`
```
Comment (Mariangela) [inline en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md:193]:
```suggestion
3. Click and expand **Local and Outbound Authentication Configuration**, select Identity Provider as the **Federated Authentication** and tick **Assert identity using mapped local subject identifier**.
```

**Comment by Tharsanan1** [comment]
@Mariangela committed the suggested changes.

File: en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md [hunk 9]
```diff
@@ -110,7 +122,21 @@ Similarly, add the Identity Server as an identity provider configurations in `ht
...
+    1. Tick `Define Custom Claim Dialect` radio box.
+    2. Click `Add Claim Mapping` and add the following data.
+        * Identity Provider Claim URI: `http://wso2.org/claims/groups`
+        * Local Claim URI: `http://wso2.org/claims/roles`
+    3. In the dropdown next to the **Role Claim URI** label, select `http://wso2.org/claims/groups`
+
+6.  Click and expand the **Role Configuration** section.
+    1. Click **Add role Mapping**.
+    2. Add the following row 
+        * Identity Provider Role: `Internal/creator`
```
Comment (Mariangela) [inline en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md:135]: @Tharsanan1 , what do you mean by this line?

"Add the following row "

Comment (Tharsanan1) [reply en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md:135]: After we click `Add role Mapping` we get a table. in the table we should give `Internal/creator` for Identity Provider Role column

File: en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md [hunk 10]
```diff
@@ -110,7 +122,21 @@ Similarly, add the Identity Server as an identity provider configurations in `ht
...
+5.  Click and expand the **Claim Configuration** section, and click and expand **Basic Claim Configuration**.
+    1. Tick `Define Custom Claim Dialect` radio box.
+    2. Click `Add Claim Mapping` and add the following data.
+        * Identity Provider Claim URI: `http://wso2.org/claims/groups`
+        * Local Claim URI: `http://wso2.org/claims/roles`
+    3. In the dropdown next to the **Role Claim URI** label, select `http://wso2.org/claims/groups`
+
+6.  Click and expand the **Role Configuration** section.
+    1. Click **Add role Mapping**.
+    2. Add the following row
```
Comment (Tharsanan1) [inline en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md:134]:
```suggestion
    2. Create a table entry with the following data
```

File: en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md [hunk 11]
```diff
@@ -110,7 +122,21 @@ Similarly, add the Identity Server as an identity provider configurations in `ht
...
+5.  Click and expand the **Claim Configuration** section, and click and expand **Basic Claim Configuration**.
+    1. Tick `Define Custom Claim Dialect` radio box.
+    2. Click `Add Claim Mapping` and add the following data.
+        * Identity Provider Claim URI: `http://wso2.org/claims/groups`
+        * Local Claim URI: `http://wso2.org/claims/roles`
+    3. In the dropdown next to the **Role Claim URI** label, select `http://wso2.org/claims/groups`
+
+6.  Click and expand the **Role Configuration** section.
+    1. Click **Add role Mapping**.
+    2. Create a table entry with the following data
```
Comment (Mariangela) [inline en/docs/reference/customize-product/extending-api-manager/saml2-sso/configuring-identity-server-as-idp-for-sso.md:134]:
```suggestion
    2. Create a table entry with the following data.
```

**Review by Mariangela — APPROVED** [review]

**Comment by Mariangela** [comment]
@Tharsanan1 , please send these changes to the 4.2.0 branch as well.

**Pull request merged**